docs/getting-started/tutorial-visualizing.asciidoc

```diff
@@ -154,15 +157,17 @@ image::images/tutorial-visualize-map-2.png[]
 
 . Navigate the map by clicking and dragging.  Use the controls
 on the left to zoom the map and set filters.
-. *Save* this map with the name `Map Example`.
+. In the application toolbar, click *Save*.
```

Comment:
Elsewhere in the tutorial, we shorten this a bit.  For example:

To save this chart so you can use it later, click *Save* in
the top menu bar and enter `Pie Example`.